src/buildercore/bigquery.py

```diff
@@ -1,4 +1,34 @@
-BIGQUERY_SCHEMAS_FOLDER = 'src/buildercore/bigquery/schemas'
+from . import config
+import os, requests, logging
+from os.path import join
 
-def schema(schema_name):
-    return open('%s/%s.json' % (BIGQUERY_SCHEMAS_FOLDER, schema_name))
+LOG = logging.getLogger(__name__)
+
+def local_schema_path(stackname, schema_name):
+    "returns a local path to the given `schema_name`"
+    # "/.../.cfn/bigquery-schema/elife-data-pipeline--prod/fooschema.json"
+    return join(config.BIGQUERY_SCHEMA_DIR, stackname, schema_name)
+
+def fetch_remote_schema(stackname, file_url):
```

Comment:
should use [the Terraform HTTP data source](https://www.terraform.io/docs/providers/http/data_source.html) to avoid hand-rolling the download, we use it already [for HTML error pages](https://github.com/elifesciences/builder/blob/master/src/buildercore/terraform.py#L289-L291)

Reply:
good call